src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/offendersearch/MatchRequest.java

```diff
@@ -71,9 +72,9 @@ public MatchRequest buildFrom(String pnc, uk.gov.justice.probation.courtcasematc
         }
 
         public MatchRequest buildFrom(CourtCase courtCase) throws IllegalArgumentException {
-            uk.gov.justice.probation.courtcasematcher.model.domain.Name defendantName = Optional.ofNullable(courtCase.getName())
-                    .orElseGet(() -> nameHelper.getNameFromFields(courtCase.getDefendantName()));
-            return buildFrom(courtCase.getPnc(), defendantName, courtCase.getDefendantDob());
+            final var defendantName = Optional.ofNullable(courtCase.getFirstDefendant().getName())
```

Comment:
How can this be null after validation ?

Reply:
Good point, this method's gotten a bit mangled. Will have a look and tidy up.